docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,176 @@
+The configuration store utilizes the opensource project https://github.com/TechnologyBrewery/krausening[Krausening,role=external,window=_blank] for managing its configurations, where each configuration is 
+represented by a key value pair property. The key values pairs are stored as string types using the standard `.properties` file 
+format.
+
+Here is an example properties file:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=baseTopic
+----
+This file defines two configuration properties: `connector` and `topic`.
+
+TIP: For sensitive configuration values, Krausening supports https://github.com/TechnologyBrewery/krausening/blob/dev/krausening/README.md#krausening-in-four-pints-leveraging-jasypt-for-encryptingdecrypting-properties[encrypting and decrypting properties,role=external,window=_blank]
+
+==== Environment Specific Overrides
+In addition to the properties file introduced above, one can optionally specify a secondary properties file that houses environment
+specific configurations. These configurations will then used to override their respective base configurations.
+
+Continuing with the example above, suppose one defines an additional properties file like the following:
+[source,properties]
+----
+topic=ciTopic
+auth-token=auth-secret
+----
+
+Then when these two properties files are reconciled within the store, it results in a configuration equivalent to the following:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=ciTopic
+auth-token=auth-secret
+----
+
+NOTE: The `topic` property is overwritten when the environment specific value `ciTopic`.
+
+=== Deploying the Configuration Store
+The https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md[configuration store helm chart,role=external,window=_blank] is used for easily deploying it to a Kubernetes cluster. As part of the deployment, one must
+<<mounting-properties-files,mount the properties files>> to the Kubernetes resource allowing the configuration store to access the custom properties. Additionally,
+the <<backend-storage-types,backend storage type>> can be adjusted to fit a project's needs.
+
+==== Mounting Properties Files
+Properties files are attached to the local file system of the configuration store within Kubernetes through a mounted volume. To mount
+a volume to the configuration store, one must set the helm chart's https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md#persistent-volume-properties[persistent volume properties,role=external,window=_blank]. Then the helm chart's https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md#environment-variables[environment 
+variables,role=external,window=_blank] must be updated with the corresponding URI(s) to the properties files within that volume. It is required to specify 
```

Comment:
A: I think it might be better to use the word "location" over "URI", since (at least for now) we are _assuming_ the URI will be a local `file://` URI.